packages/zowe-explorer/src/uss/ZoweUSSNode.ts

```diff
@@ -171,7 +171,9 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
                 this.label,
                 localize("getChildren.error.response", "Retrieving response from ") + `uss-file-list`
             );
-            await refreshTree(sessNode);
+            await syncSessionNode(Profiles.getInstance())((profileValue) =>
+                ZoweExplorerApiRegister.getUssApi(profileValue).getSession()
```

Comment:
I am leery of passing `getSession` in as a param. We have some circular dependency problem in ZE weith the API functions. The API imports functions from throughout ZE, so if ZE has to import API functions, we might get random test failures (this happened when we added base profile support awhile back).

This seems like a structural issue that we should really deal with at some point. Not in this PR or anything. But you've reminded me of it. I'll bring it up at the next standup probably...

Reply:
I do not see any circular dependency issues here, because we are passing function as an argument, it is not a class or a module, it has no context.

But you are right, with passing classes and objects, there might be such a problem :(